{"uid":"b338f29e6a5c6c8f","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1702602867013,"stop":1702602877259,"duration":10246},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564af4866d33 <unknown>\n#1 0x564af4523f87 <unknown>\n#2 0x564af4573c0e <unknown>\n#3 0x564af4573d11 <unknown>\n#4 0x564af45bf004 <unknown>\n#5 0x564af459d36d <unknown>\n#6 0x564af45bc297 <unknown>\n#7 0x564af459d0e3 <unknown>\n#8 0x564af4565044 <unknown>\n#9 0x564af456644e <unknown>\n#10 0x564af482b861 <unknown>\n#11 0x564af482f785 <unknown>\n#12 0x564af4819285 <unknown>\n#13 0x564af483041f <unknown>\n#14 0x564af47fd20f <unknown>\n#15 0x564af4854028 <unknown>\n#16 0x564af48541f7 <unknown>\n#17 0x564af4865ed4 <unknown>\n#18 0x7f0fb8094ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f7d3864d410>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"d32c5388af45d1a4f37d95a917d396dc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f7d386edf80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564af4866d33 <unknown>\nE       #1 0x564af4523f87 <unknown>\nE       #2 0x564af4573c0e <unknown>\nE       #3 0x564af4573d11 <unknown>\nE       #4 0x564af45bf004 <unknown>\nE       #5 0x564af459d36d <unknown>\nE       #6 0x564af45bc297 <unknown>\nE       #7 0x564af459d0e3 <unknown>\nE       #8 0x564af4565044 <unknown>\nE       #9 0x564af456644e <unknown>\nE       #10 0x564af482b861 <unknown>\nE       #11 0x564af482f785 <unknown>\nE       #12 0x564af4819285 <unknown>\nE       #13 0x564af483041f <unknown>\nE       #14 0x564af47fd20f <unknown>\nE       #15 0x564af4854028 <unknown>\nE       #16 0x564af48541f7 <unknown>\nE       #17 0x564af4865ed4 <unknown>\nE       #18 0x7f0fb8094ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1702602827228,"stop":1702602827228,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1702602827228,"stop":1702602827228,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1702602827229,"stop":1702602827229,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564af4866d33 <unknown>\n#1 0x564af4523f87 <unknown>\n#2 0x564af4573c0e <unknown>\n#3 0x564af4573d11 <unknown>\n#4 0x564af45bf004 <unknown>\n#5 0x564af459d36d <unknown>\n#6 0x564af45bc297 <unknown>\n#7 0x564af459d0e3 <unknown>\n#8 0x564af4565044 <unknown>\n#9 0x564af456644e <unknown>\n#10 0x564af482b861 <unknown>\n#11 0x564af482f785 <unknown>\n#12 0x564af4819285 <unknown>\n#13 0x564af483041f <unknown>\n#14 0x564af47fd20f <unknown>\n#15 0x564af4854028 <unknown>\n#16 0x564af48541f7 <unknown>\n#17 0x564af4865ed4 <unknown>\n#18 0x7f0fb8094ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f7d3864d410>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"d32c5388af45d1a4f37d95a917d396dc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f7d386edf80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564af4866d33 <unknown>\nE       #1 0x564af4523f87 <unknown>\nE       #2 0x564af4573c0e <unknown>\nE       #3 0x564af4573d11 <unknown>\nE       #4 0x564af45bf004 <unknown>\nE       #5 0x564af459d36d <unknown>\nE       #6 0x564af45bc297 <unknown>\nE       #7 0x564af459d0e3 <unknown>\nE       #8 0x564af4565044 <unknown>\nE       #9 0x564af456644e <unknown>\nE       #10 0x564af482b861 <unknown>\nE       #11 0x564af482f785 <unknown>\nE       #12 0x564af4819285 <unknown>\nE       #13 0x564af483041f <unknown>\nE       #14 0x564af47fd20f <unknown>\nE       #15 0x564af4854028 <unknown>\nE       #16 0x564af48541f7 <unknown>\nE       #17 0x564af4865ed4 <unknown>\nE       #18 0x7f0fb8094ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1702602867013,"stop":1702602877259,"duration":10246},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564af4866d33 <unknown>\n#1 0x564af4523f87 <unknown>\n#2 0x564af4573c0e <unknown>\n#3 0x564af4573d11 <unknown>\n#4 0x564af45bf004 <unknown>\n#5 0x564af459d36d <unknown>\n#6 0x564af45bc297 <unknown>\n#7 0x564af459d0e3 <unknown>\n#8 0x564af4565044 <unknown>\n#9 0x564af456644e <unknown>\n#10 0x564af482b861 <unknown>\n#11 0x564af482f785 <unknown>\n#12 0x564af4819285 <unknown>\n#13 0x564af483041f <unknown>\n#14 0x564af47fd20f <unknown>\n#15 0x564af4854028 <unknown>\n#16 0x564af48541f7 <unknown>\n#17 0x564af4865ed4 <unknown>\n#18 0x7f0fb8094ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"2769fbbe75df89dd","name":"public ip address","source":"2769fbbe75df89dd.txt","type":"text/plain","size":13},{"uid":"1733ca9a154169ac","name":"Cookies","source":"1733ca9a154169ac.json","type":"application/json","size":3358},{"uid":"d60a7edabd5673","name":"Session Storage","source":"d60a7edabd5673.json","type":"application/json","size":358},{"uid":"bc7a333c5dc7fa7f","name":"Local Storage","source":"bc7a333c5dc7fa7f.json","type":"application/json","size":208},{"uid":"7e2e6b437c281629","name":"Console Logs","source":"7e2e6b437c281629.json","type":"application/json","size":2},{"uid":"cb2c2914e177577f","name":"Full Page Screenshot","source":"cb2c2914e177577f.png","type":"image/png","size":1280118},{"uid":"97f9f9033a9d7edf","name":"URL","source":"97f9f9033a9d7edf.uri","type":"text/uri-list","size":23},{"uid":"a621ffe0bc961616","name":"Network Logs","source":"a621ffe0bc961616.json","type":"application/json","size":25812}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1538-860"},{"name":"thread","value":"2305-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"b338f29e6a5c6c8f.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}